sdks/python/apache_beam/dataframe/frame_base.py

```diff
@@ -100,9 +100,7 @@ def __repr__(self):
 
 
 class DeferredFrame(DeferredBase):
-  @property
-  def dtypes(self):
-    return self._expr.proxy().dtypes
```

Comment:
@robertwb do you know why we had dtypes defined in `DeferredFrame`, is this just because it predates `DeferredFrameOrSeries`?

Reply:
Pretty sure it's just because it predated DeferredDataFrameOrSeries. Though at this point, DeferredFrame doesn't seem to serve much use at all.

Reply:
Good point we should probably just drop it and reference `DeferredBase.wrap`. I'll save that for another PR to keep the diff clean here.